client/state/domains/management/validation-schemas/reducer.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { DOMAIN_MANAGEMENT_VALIDATION_SCHEMA_ADD } from 'state/action-types';
+
+export const validationSchemas = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case DOMAIN_MANAGEMENT_VALIDATION_SCHEMA_ADD:
```

Comment:
can we add some kind of comment or explanation here of what `action.schemas` is expected to look like?

Reply:
✅